### ISSUE TITLE
请教您一个关于语义分割训练的问题

### ISSUE THREAD

**Issue by zouzhekang** [bubbliiiing/Semantic-Segmentation#5]
大佬，我关注您的CSDN后发现您也有在学tf2.0，我想问问语义分割是怎么训练的呀？怎么确定它没有过拟合？
比如network.compile(optimizer = optimizers.Adam(lr = 0.01)loss = ‘MSE’,metrics = [‘Meaniou’])，请问那个metrics指的是什么啊？怎么每个step都有变化？我是不是得每fit完一个epoch就算一次mIOU才不会过拟合？
大佬，我star你了，求帮忙，嘤嘤

**Comment by bubbliiiing** [comment]
你有个验证集用于验证就行了，也不需要每次都算miou吧
